src/job/manifestJobHandler.ts

```diff
@@ -40,16 +43,47 @@ export class ManifestJobHandler extends JobHandler {
 
   // TODO: Make this a non-state-mutating function, e.g. return the deployCommands?
   prepDeployCommands(): void {
+    const b = this._config.get<string>('searchIndexBucket') ?? 'docs-search-indexes-test';
+    // /deploy -> send to /prd folder. /test-deploy -> send to /preprd folder
+    const env = this._config.get<string>('env');
+    // Note: mut-index automatically prepends 'search-indexes/' to the folder.
+    const f = this._config.get<string>('searchIndexFolder')[env] ?? 'fallback-folder';
+    this.logger.info(this.currJob._id, `Manifest attempt to upload to bucket: ${b}, folder: ${f}`);
+    // Due to the dual existence of prefixes, check for both for redundancy
+    const maP = this.currJob.manifestPrefix ?? this.currJob.payload.manifestPrefix;
+    const muP = this.currJob.mutPrefix ?? this.currJob.payload.mutPrefix;
+    const url = this.currJob.payload.url;
```

Comment:
nit: totally optional but i'd suggest more indicative variables names for `b` `f` `maP` and `muP`

Reply:
Haha, yes, this probably could be better... I went this route to keep the actual `mut-index` command somewhat condensed. Going to leave it